pandas/core/arrays/arrow/array.py

```diff
@@ -444,7 +444,7 @@ def dropna(self: ArrowExtensionArrayT) -> ArrowExtensionArrayT:
         else:
             return type(self)(pc.drop_null(self._data))
 
-    def isin(self: ArrowExtensionArrayT, values) -> npt.NDArray[np.bool_]:
+    def isin(self, values) -> npt.NDArray[np.bool_]:
```

Comment:
Why make this change?

Reply:
This TypeVar is useless (my local pyright version complained about it). I'll bump pyright on the CI